### PR TITLE
Item admin mange user account mary

### PR DESCRIPTION
**Pull request by picka18**
Admin Generate Report Page
Edit Admin Generate Reports Card
Update Heath Prof Page
View Health Prof Page

3-16-2024

### CODE REVIEW

File: app/src/main/res/layout/fragment_admin_generate_reports_page.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:layout_marginTop="16dp"
+            android:layout_marginStart="8dp"
+            android:calendarViewShown="false"
+            android:datePickerMode="spinner"/>
+
+    </LinearLayout>
+
+    <LinearLayout
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_admin_generate_reports_page.xml:54]: This Linear Layout, overlaps the top linear layout, please fix

File: app/src/main/res/layout/fragment_admin_generate_reports_page.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        android:id="@+id/title_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Generate Reports"
+        android:textSize="24sp"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="16dp"
+        android:textStyle="bold" />
+
+    <Spinner
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_admin_generate_reports_page.xml:19]: This one overlaps as well, please rearrange propely

File: app/src/main/res/layout/fragment_update_health_prof_page.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        android:layout_marginBottom="24dp"/>
+
+    <EditText
+        android:id="@+id/editTextHealthProfName"
+        android:layout_width="383dp"
+        android:layout_height="57dp"
+        android:hint="Health Professional Name"
+        android:inputType="text" />
+
+    <TextView
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_update_health_prof_page.xml:25]: This portion was supposed to show values of non editable parts, please rewrite to contain a textview for label and value, to make it easier for devs to edit from backend.

File: app/src/main/res/layout/fragment_update_health_prof_page.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        android:textSize="16sp" />
+
+    <EditText
+        android:id="@+id/editTextHealthProfSpecialty"
+        android:layout_width="380dp"
+        android:layout_height="55dp"
+        android:hint="Specialty"
+        android:inputType="text" />
+
+    <TextView
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_update_health_prof_page.xml:45]: This portion was supposed to show values of non editable parts, please rewrite to contain a textview for label and value, to make it easier for devs to edit from backend.

File: app/src/main/res/layout/fragment_view_health_prof_page.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_view_health_prof_page.xml:1]: Please rewrite as labels and value to present information. Also missing other values for health professional. Where does specialty come from? There's no specialty input in the create health prof page.

File: app/src/main/res/layout/fragment_update_health_prof_page.xml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        android:background="@android:color/white"
+        android:clickable="false"
+        android:focusable="false"
+        android:longClickable="false"
+        android:padding="16dp"
+        android:text="This is a non-editable text"
+        android:textColor="@android:color/black"
+        android:textSize="16sp" />
+
+    <EditText
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_update_health_prof_page.xml:38]: Missing other values for health professional. Where does specialty come from? There's no specialty input in the create health prof page.

File: app/src/main/res/layout/fragment_admin_generate_reports_page.xml (hunk 1 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_admin_generate_reports_page.xml:54]: done fix overlaps top linear layout

File: app/src/main/res/layout/fragment_admin_generate_reports_page.xml (hunk 2 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_admin_generate_reports_page.xml:19]: dine fix overlaps

File: app/src/main/res/layout/fragment_update_health_prof_page.xml (hunk 4 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_update_health_prof_page.xml:45]: done rewrite into textview

File: app/src/main/res/layout/fragment_update_health_prof_page.xml (hunk 3 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_update_health_prof_page.xml:25]: done

File: app/src/main/res/layout/fragment_view_health_prof_page.xml (hunk 5 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_view_health_prof_page.xml:1]: done change

File: app/src/main/res/layout/fragment_update_health_prof_page.xml (hunk 6 above)
Comment (picka18) [reply app/src/main/res/layout/fragment_update_health_prof_page.xml:38]: done change and create health prof page

**Comment by RevyaS** [comment]
Approved

**Pull request merged**